docs/markdown/sbt-setup.md

```diff
@@ -0,0 +1,57 @@
+---
+sidebar_position: 3
+---
+
+# SBT setup
+
+You can enable `snapshot4s` for a variety of SBT project layouts.
+
+ - For single project SBT builds follow [these instructions](#single-project-builds).
+ - For multi-project SBT builds follow [these instructions](#multi-project-builds)
+ - For `sbt-projectmatrix` follow [these instructions](#sbt-projectmatrix).
+
+snapshot4s does not yet support `sbt-crossproject` or `sbt-typelevel`.
```

Comment:
Should we do something like 
```suggestion
snapshot4s does not yet support `sbt-crossproject` or `sbt-typelevel` but it is on our roadmap #123 #123.
```
Where the `#123` would be the issue numbers?

Reply:
I've opened https://github.com/siriusxm/snapshot4s/issues/28.